test/hotspot/jtreg/vmTestbase/nsk/jvmti/GetClassStatus/getclstat007/getclstat007.cpp

```diff
@@ -25,7 +25,7 @@
 #include <string.h>
 #include "jvmti.h"
 #include "agent_common.hpp"
-#include "JVMTITools.h"
+#include "JVMTITools.hpp"
```

Comment:
There's a lower case jvmti_tools.hpp and an uppercase JVMTITools.hpp now.  Maybe someone could do a cleanup of these tests (please!)